### DISCUSSION TITLE
Abandoned Ice Mine - New Ruin / Outpost Explorable ?

### DISCUSSION THREAD

**Discussion by Scuba-Recon** [FakeFishGames/Barotrauma#12724] [Ideas and suggestions]
Most of the mining props & assets haven't been used too much in-game except for a brief small area on outposts. 

To make more use of them and have a new expanded environment to optionally explore or useful for a mission type tasks. Either it can be designed initially using a outpost area the sub docks onto to venture around and explore. 

Or having it as an alternative for alien ruins which could generate in caverns which similar to beacon stations have an airlock area to safely board (a docking hatch too if the sub or shuttle has one of those to utilize) 

The derelict/abandoned mine compared to the ruins could have a partially safe environment with dry areas and barely functioning o2 generators/pumps (optional task perhaps to power them up so as to explore without need of Diving Suits). There could be additional hazards such as radiation from uranium deposits throughout the mine. Small ambush triggers of husks and crawler/raptors which carve their way out of inaccessible doors, ice walls or ceilings or even some armed pirates attempting to loot the mine like the player is. Random spawned deposits of common/uncommon ore types depending on the size of the mine and more generated on higher difficulty levels. 

Missions could be varied from using the current [Mining "Ore"] Mission which the ore is spawned in a generated Ice Mine on chance rather than in a deep cavern tunnel. A rescue mission which the team has to dig through a collapsed part of the mine which survivors are trapped and have them board the submarine (easier using a docked hatch). The collapsed wall in question could require an easy route quest of getting the power back to a drill which would clear the wall, or having to use multiple plasma cutters to cut through the collapsed area (takes longer but a backup alternative if for some reason the drill is bugged or didn't work so as to still complete the task). 

![Ice Mine Drill Quest](https://github.com/Regalis11/Barotrauma/assets/101093085/8e4c2e7c-9a57-4912-8451-1db488516a7c)

Basically it's ruin exploration with something of human design to something other than a submarine wreck, but abandoned or lost from unknown reasons which you'd finds remains of other explorers and dead corpses with mining outfits. The outer areas of the Ice mine could use thin shell walls for those inclined to breach through using explosives or cutters, while the inner areas or critical parts would use an indestructible heavier shell similar to the alien setup. (Another potential backstory is during the mining operation in the cavern they accidentally thawed out a husk parasite nest and became infected leading to the mine being abandoned)

I can probably update this with more concepts such as that drill one above if more context of areas shown are helpful, I'm only good with partially building  subs, but outpost/ruin design in the editor isn't something I specialize with.

**Comment by Hazmat616** [comment]
Even more mines are prone to mudraptor infestations like that one event. So having a completely flooded mine serve as nests could work too

**Comment by duarm** [comment]
There's a mod which adds dry caves. Custom handmade outposts, cave-themed dungeons which are big and not underwater. Not sure if the mod adds dry caves in place of normal underwater caves, but this should be added as a mission type.
